src/api/authorization/index.ts

```diff
@@ -1,2 +1,6 @@
 export { authorizationApi } from './authorization.api';
-export type { LoginFormData, RegistrationFormData } from './types';
+export type {
+  LoginFormData,
+  RegistrationFormData,
+  OauthTokenData,
+} from './types';
```

Comment:
```suggestion
  OauthTokenData,
  AuthorizationModel,
} from './types';
```

Все, что используется вне модуля, реэкспортируем "наружу".
`index.ts` - это интерфейс модуля, при импорте дальше него проваливаться в файлы нежелательно